.gitignore

```diff
@@ -49,6 +49,7 @@
 /resources/frontend_client/embed.html
 /resources/frontend_client/index.html
 /resources/frontend_client/public.html
+/resources/metabase/*.edn
```

Comment:
this seems like a really big gitignore. should this be a bit more targeted?

Reply:
i'm wondering if we add more edn config files, this would be the normal place they might go. Perhaps add a directory `resources/metabase/i18n` or translate or something that things can go into.

Reply:
yeah that's a good point. That was the old directory for the resource bundle class files so I didn't change it